### DISCUSSION TITLE
Welcome to WEIS Discussions!

### DISCUSSION THREAD

**Discussion by rafmudaf** [NLRWindSystems/WEIS#242] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mlidGWT** [comment]
Hi fam! 👋
My name is Meg and I work at a rotor technology company. I am working on using WEIS in my day job to generate and run DLCs to help us design and analyze cool stuff! I do not consider myself a skilled coder/computer user, so probably won't be making any direct contributions to the codebase, but I hope to suggest features and improvements to help make WEIS more user friendly. I also have a skill for finding the edge cases not supported by existing code, but not knowing how to solve them, so you can be sure to find me in the Issues pages. 😂
